Extensions/AzureExp/Src/Tasks/ManageExperimentV1/experimentmanager.ts

```diff
@@ -50,7 +51,10 @@ export default class ExperimentManager {
         tl.debug(JSON.stringify(response));
     }
 
-    public async getExperiment(experimentName: string): Promise<any> {
+    /**
+     * @param experimentName filter by experiment name
+     */
+    public async getExperiments(experimentName?: string): Promise<any[]> {
         let requestUrl = `https://exp.microsoft.com/api/features/${this._featureId}/progressions/${this._progressionId}`;
```

Comment:
Write some console logs like: Getting the experiment for the feature/progression